test/components/extractors/test_named_entity_extractor.py

```diff
@@ -21,15 +22,17 @@ def test_named_entity_extractor_backend():
 @pytest.mark.unit
 def test_named_entity_extractor_serde():
     extractor = NamedEntityExtractor(
-        backend=NamedEntityExtractorBackend.HUGGING_FACE, model_name_or_path="dslim/bert-base-NER", device_id=-1
+        backend=NamedEntityExtractorBackend.HUGGING_FACE,
+        model_name_or_path="dslim/bert-base-NER",
+        device=ComponentDevice.from_str("cuda:22"),
```

Comment:
I'm curious about `cuda:22`, is it representing a non-existing device or something specific? In either case I'd add a small comment to clarify.

Reply:
The ID is arbitrary - it can be anything. I can change it to `1` to make it make it look less suspicious 😉

Reply:
Ahah thank you!